UI/BuildDesignatorWnd.cpp

```diff
@@ -1059,7 +1052,7 @@ void BuildDesignatorWnd::CenterOnBuild(int queue_idx, bool open) {
         if (std::shared_ptr<const UniverseObject> build_location = objects.Object(location_id)) {
             // centre map on system of build location
             int system_id = build_location->SystemID();
-            MapWnd* map = ClientUI::GetClientUI()->GetMapWnd();
+            auto&& map = ClientUI::GetClientUI()->GetMapWnd();
```

Comment:
why `auto&&` here? Other uses are mostly `const auto& map_wnd` so why not `auto& map_wnd`? Is `auto&&` more appropriate when not const...?